doc/rbd/rbd-mirroring.rst

```diff
@@ -305,10 +305,9 @@ For example::
 .. tip:: ``rbd-mirror`` tunables are set by default to values suitable for
    mirroring an entire pool.  When using ``rbd-mirror`` to migrate single
    volumes been clusters you may achieve substantial performance gains
-   by setting ``rbd_mirror_journal_max_fetch_bytes=33554432`` and
-   ``rbd_journal_max_payload_bytes=8388608`` within the ``[client]`` config
-   section of the local or centralized configuration.  Note that these
-   settings may allow ``rbd-mirror`` to present a substantial write workload
+   by setting ``rbd_journal_max_payload_bytes=8388608`` within the ``[client]``
+   config section of the local or centralized configuration.  Note that this
+   setting may allow ``rbd-mirror`` to present a substantial write workload
```

Comment:
I am not sure about achieving "substantial performance gains" with just increasing `rbd_journal_max_payload_bytes`. I suppose `max_fetch_bytes` was more important here and now it depends on  the memory target.

So I am not sure this tip will be very useful now unless someone try it and confirm there is still a visible gain. And it will be useful only for large writes.

Also it would be nice to have a tip about rbd_mirror_memory_target, which may affect replay performance considerably.

Reply:
> Also it would be nice to have a tip about rbd_mirror_memory_target, which may affect replay performance considerably.

@zdover23 ^^ for (the bottom of) your TODO list

Reply:
At the time, they were suggested to me as a pair.  I did not evaluate the effects of setting them individually in isolation.  I was migrating volumes between clusters for capacity management, one or two at a time, and needed the transfers to complete as quickly as possible.  With the defaults, attached volumes often would take an hour or two to catch up, or in some cases would fall increasingly behind.